src/test/components/__snapshots__/TrackNetwork.test.js.snap

```diff
@@ -702,7 +702,7 @@ exports[`timeline/TrackNetwork matches the component snapshot 1`] = `
       class="timelineVerticalIndicatorsLine"
       data-marker-index="10"
       data-testid="vertical-indicator-line"
-      style="--vertical-indicator-color: var(--red-60); left: 66.66666666666667px;"
+      style="--vertical-indicator-color: var(--red-60); left: 200px;"
```

Comment:
Did you expect this change?

Reply:
Yes, this changes the duration markers' indicator from the start to the end.